mantidproject/mantid
# Re #23690 Fixed issue with overwriting focus results

**Description of work.**
Altered focus to prevent overwriting of its results when it needed to crop them due to a problematic vanadium spline
**To test:**
This test will only work with access to the ISIS Data Search Archive
run the following script with these [files](https://github.com/mantidproject/mantid/files/2438244/GemTest.zip)
(Ensure you update the paths)
```
from isis_powder import Gem

config_file_path = r"Path/to/Gem_config_example.yaml"
Gem_example = Gem(config_file=config_file_path,user_name="test")
Gem_example.create_vanadium(input_mode="Individual",first_cycle_run_no=85374)
Gem_example.focus(input_mode="Individual", run_number="85374")
```
Then check in either ResultsD or ResultsTOF the workspaces within these groups should not all be identical.
Fixes #23690 

**High priority as a user is relying on this being in the nightly build**

*This does not require release notes as it was the bug does not exist in any release builds*

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
